docker/compilers/gcc/Dockerfile

```diff
@@ -53,38 +65,54 @@ RUN /gcc-$GCC_VERSION/configure \
         --enable-cet \
         --disable-multilib \
         --without-cuda-driver \
-        --enable-checking=release \
-    && make -j "$(nproc)" \
-    && make install-strip DESTDIR=/gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION \
+        --enable-checking=release
+
+RUN make -j "$(nproc)"
+
+RUN make install-strip DESTDIR=/gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION \
     && mkdir -p /gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION/usr/share/gdb/auto-load/usr/lib64 \
-    && mv /gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION/usr/lib64/libstdc++.so.6.0.30-gdb.py /gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION/usr/share/gdb/auto-load/usr/lib64/libstdc++.so.6.0.30-gdb.py
+    && mv \
+        /gcc-$GCC_VERSION-$BUILD_VERSION-ubuntu-$UBUNTU_VERSION/usr/lib64/libstdc++.so.6.0.30-gdb.py \
```

Comment:
This probably will change with newer versions of gcc? should we have this as arg too then?

Reply:
Honestly, I don't know. If it's going to change, I will put it in a variable when updating the image